src/feature/control/control_getinfo.c

```diff
@@ -326,46 +326,37 @@ getinfo_helper_current_time(control_connection_t *control_conn,
 }
 
 /**
- * Switch between microdesc vs networkstatus descriptor dumps
- * Assumes that question would be either:
- * - dir/status-vote/current/consensus-microdesc
- * or
- * - dir/status-vote/current/consensus
+ * Helps to access directory information for a given
```

Comment:
This still doesn't look like our usual documentation format. Please describe the return value in a proper sentence and not in a list.